Package.swift

```diff
@@ -13,6 +13,12 @@ import PackageDescription
 
 let package = Package(
   name: "SwiftProtobuf",
+  platforms: [
+      .iOS(.v10),
```

Comment:
Yup. I don't think it's worth messing with the 1.x branch at the moment, I'm hoping we can sunset it in the near future.

Should I tweak the one `.md` to call out if the new Xcode raises a deployment minimum that that forces a major version change on the library? Just so we always remember that.

Reply:
I don't think we want to do this. I think we should just remove the part about the minion deployment version and only consider the Swift version.

Reply:
I don't think we can leave it out of the `.podspec`, but I'll drop it here.

Reply:
Updated.

Reply:
> I don't think we can leave it out of the `.podspec`, but I'll drop it here.

I tried removing the deployment attributes from the `.podspec`, and `pod lib lint` fails, from the errors, CocoaPods must use some hard coded values vs. getting some defaults from Xcode (atleast I'd hope Xcode didn't try to default to macOS 10.6 and iOS 8.0 since Xcode 14.3.1 can't really build for those). So it looks like we do need to keep the values in the `.podspec`.